convert-to-lowercase/README.md

```diff
@@ -0,0 +1,41 @@
+# Convert Files and References to Lowercase
+
+This shell script converts all filenames in a folder (and subfolders) to lowercase. Additionally, any references to the file will be replaced.
```

Comment:
References to the file located where? In other files in that directory? I am guessing this doesn't comb through the whole world.